common/models/user.js

```diff
@@ -1,3 +1,33 @@
+var mailerService = require('../../server/server-utils/emailProvider');
+
 module.exports = function(user) {
+    var User = user;
+
+
+    User.on('resetPasswordRequest', function(info){
+      var MailService = new mailerService();
+      var accessToken = info.accessToken.id;
+      var email = info.email;
+      var user_id = info.accessToken.userId;
+
+      var resetURL = 'http://localhost:8080/resetPassword/?id=' + user_id + '&token=' + accessToken;
```

Comment:
Oh dear god, how did I miss that... and once it's on github, it's here forever. I may need to change my name and move states.